### ISSUE TITLE
UnicodeEncodeError

### ISSUE THREAD

**Issue by madscort** [chklovski/CheckM2#55]
Hi,

I use CheckM2 (v. 1.0.1) on a HPC cluster and I get this error whenever I run CheckM2 as a submitted job (qsub):

`Traceback (most recent call last):
  File "/services/tools/checkm2/1.0.1/bin/checkm2", line 211, in <module>
    args.resume, args.remove_intermediates, args.ttable)
  File "/services/tools/checkm2/1.0.1/lib/python3.6/site-packages/checkm2/predictQuality.py", line 135, in prediction_wf
    diamond_out = diamond_search.run(prodigal_files)
  File "/services/tools/checkm2/1.0.1/lib/python3.6/site-packages/checkm2/diamond.py", line 119, in run
    self.__call_diamond(protein_chunks, diamond_out)
  File "/services/tools/checkm2/1.0.1/lib/python3.6/site-packages/checkm2/diamond.py", line 74, in __call_diamond
    sequenceClasses.SeqReader().write_fasta(seq_object, temp_diamond_input.name)
  File "/services/tools/checkm2/1.0.1/lib/python3.6/site-packages/checkm2/sequenceClasses.py", line 104, in write_fasta
    fout.write('>' + seqId + '\n')
UnicodeEncodeError: 'ascii' codec can't encode character '\u03a9' in position 18: ordinal not in range(128)`

Have you seen this before?

The cluster uses environment modules which contains the tool and all its dependencies. 
I have no problem running it on an interactive node with the exact same module loaded.

Thanks!

**Comment by madscort** [comment]
I have found the cause of the error. When jobs are submitted to the cluster, the locale on the compute node is set to "POSIX", when logging on to an interactive node, the locale is set to "en_US.utf8". Forcing it to "en_US.utf8" on the compute node solves the error for submitted jobs. I am not sure if this is something CheckM2 could have avoided, at least it is not a problem I have encountered before.
